README.md

```diff
@@ -109,14 +136,13 @@ it('should reject because of x', () => {
 
 #### Assertions and testing APIs
 
-`assert`, `describe`, `it`, `beforeEach` and `afterEach` are
-available in tests by default - you don't need to import
-anything for those to work.
+`assert`, `describe`, `it`, `beforeEach`, and `afterEach` are
```

Comment:
Oh no, would you prefer them to not be here?

Mom’s style guide actually doesn’t use them but I tend to for developer docs.

But consistent beats my preference, so let me know ;-)